kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -108,36 +107,39 @@
       tabsId="quizSectionTabs"
       :activeTabId="activeSection ? activeSection.section_id : ''"
     >
-      <p>{{ activeSection.section_id }}</p>
-      <!-- TODO This should be a separate component like "empty section container" or something -->
-      <div v-if="!activeQuestions.length" class="no-question-style">
-        <KGrid class="questions-list-label-row">
-          <KGridItem
-            class="right-side-heading"
-            style="padding: 0.7em 0.75em;"
+      <KGrid v-if="!activeQuestions.length" class="questions-list-label-row">
+        <KGridItem
+          class="right-side-heading"
+          style="padding: 0.7em 0.75em;"
+        >
+          <KButton
+            primary
+            :text="coreString('optionsLabel')"
           >
-            <KButton
-              primary
-              :text="coreString('optionsLabel')"
-            >
-              <template #menu>
-                <KDropdownMenu
-                  :primary="false"
-                  :disabled="false"
-                  :hasIcons="true"
-                  :options="activeSectionActions"
-                  @tab="e => (e.preventDefault() || $refs.selectAllCheckbox.focus())"
-                  @select="handleActiveSectionAction"
-                />
-              </template>
-            </KButton>
-          </KGridItem>
-        </KGrid>
+            <template #menu>
+              <KDropdownMenu
+                :primary="false"
+                :disabled="false"
+                :hasIcons="true"
+                :options="activeSectionActions"
+                @tab="e => (e.preventDefault() || $refs.selectAllCheckbox.focus())"
+                @select="handleActiveSectionAction"
+              />
+            </template>
+          </KButton>
+        </KGridItem>
+      </KGrid>
+      <!-- TODO This should be a separate component like "empty section container" or something -->
+      <div
+        v-if="!activeQuestions.length"
+        style="text-align: center; padding: 0 0 1em 0; max-width: 350px; margin: 0 auto;"
```

Comment:
`max-width: 350px` here is to accommodate the styles represented in the Figma so that the text remains at a readable length across languages as the same text in other languages may take up much more horizontal space